ui/core/src/actions/clp/index.ts

```diff
@@ -27,7 +54,10 @@ export default ({
   syncPools();
 
   // Then every transaction
-  api.SifService.onTx(syncPools);
+
+  api.SifService.onNewBlock(async () => {
```

Comment:
What's this for?

Reply:
tx pool syncing wasn't always occurring so moved to syncing every block for now.